### PR TITLE
fix: crash in activity list

### PR DESCRIPTION
**Pull request by brainbicycle**
This PR resolves [] 

### Description

Safer number check in activity list to fix a crash we are seeing:
https://artsynet.sentry.io/issues/4469752934/?project=5867225

Paired in mobile practice!

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- fix crash in activity list - mobile practice

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (fix crash in activity list - mobile practice)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f30d9b4bbc59766de2ade81dc630a8da5ac6635d
</p>

**Review by MounirDhahri — DISMISSED** [review]
thanks!

File: src/app/Scenes/Activity/ActivityList.tsx [hunk 1]
```diff
@@ -71,7 +71,13 @@ export const ActivityList: React.FC<ActivityListProps> = ({ viewer, type, me })
...
   if (notifications.length === 0) {
     // In order to center content, we need to offset the stickytabs header height
-    const headerOffset = -(headerMeasurements.height.value as number) ?? 0
+
+    let headerOffset = 0
+    if (typeof headerMeasurements.height.value === "number") {
+      headerOffset = -headerMeasurements.height.value
+    } else {
+      headerOffset = 0
+    }
```
Comment (MounirDhahri) [inline src/app/Scenes/Activity/ActivityList.tsx:80]: we probably don't need this

Comment (brainbicycle) [reply src/app/Scenes/Activity/ActivityList.tsx:80]: truu

**Pull request merged**